marketplace/charts/spinkube-azure-marketplace/values.yaml

```diff
@@ -45,6 +45,10 @@ global:
         tag: v0.14.1
         image: node-installer
         registry: ghcr.io/spinkube/containerd-shim-spin
+      kwasmOperatorTestConnectionWget:
```

Comment:
**question:** What is this for?

Reply:
Good question; this comes from the [test-connection.yaml](https://github.com/KWasm/kwasm-operator/blob/main/charts/kwasm-operator/templates/tests/test-connection.yaml) manifest.  Despite being under the `tests` section of the chart, since it references an image, it also needs to flow though `global.azure.images` to adhere to Azure Marketplace's (and `cpa verify`'s) requirements.

However, since we have to fork this chart anyways, we could also remove the test manifest altogether and thus avoid the need for this values config.